gc/base/MemorySubSpaceSemiSpace.cpp

```diff
@@ -464,28 +464,34 @@ MM_MemorySubSpaceSemiSpace::tearDown(MM_EnvironmentBase *env)
 void
 MM_MemorySubSpaceSemiSpace::flip(MM_EnvironmentBase *env, Flip_step step)
 {
+	Trc_MM_MSSSS_flip_entry(env->getLanguageVMThread());
```

Comment:
you can remove entry and exit trace points, they don't bring any useful info (as long as we report one for each entered step)